index.cjs

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+/**
+ * @typedef SocketCliJsCommands
+ * @property {import('./lib/commands/report/create.js').createReport} createReport
+ */
+
+/**
+ * @typedef SocketCliJsHelpers
+ * @property {import('./lib/utils/path-resolve.js').getPackageFiles} getPackageFiles
+ */
+
+/** @returns {Promise<SocketCliJsCommands>} */
+async function getCommands () {
+  const [
+    { createReport }
+  ] = await Promise.all([
+    import('./lib/commands/report/create.js'),
+  ])
+
+  return {
+    createReport,
```

Comment:
this needs the `import.meta` bound because `meow` needs it, so it isn't quite usable as it is.